### PR TITLE
feat: change 3rd activity card to a coming soon card

### PR DESCRIPTION
**Pull request by darren2hang**
- I made a ternary operator in the ActivityHome map function (not sure if this is the best way to go about it)
- I also made the text of the card a little longer by adding "Stay tuned!" (The width of the card is based on the content of the text, so otherwise the coming soon card is horizontally smaller than the other cards)

closes: #67

### CODE REVIEW

File: src/components/Activities.tsx [hunk 1]
```diff
@@ -18,10 +18,10 @@ import './styles/Activities.scss';
 /**
  * The following is a mapping between an activity and its descirption
  */
-const ACTIVITIES: {[key: string]: string} = {
+const ACTIVITIES: { [key: string]: string } = {
   'lost-in-translation': 'When you use the search bar, how does YouTube know what you’re looking for?',
   'sending-videos': 'Videos hold a lot of information! How can pages load quickly?',
-  'mind-reading': 'Have you ever wondered how YouTube knows what you want to watch next?',
+  'coming-soon': 'A third activity is on its way! Stay tuned!',
```
Comment (reginawang99) [inline src/components/Activities.tsx:24]: maybe we can just take this out of the map?

File: src/components/Activities.tsx [hunk 2]
```diff
@@ -34,13 +34,22 @@ function ActivityHome(props: ActivityHomeProps): JSX.Element {
...
-          </Link>,
-        )}
+        {Object.keys(ACTIVITIES).map((path, i) => {
+          return (
+            (path === 'coming-soon') ?
+              <div key={`activities-card-${i}-coming-soon`}>
+                <ActivityCard
+                  id={path}
+                  description={ACTIVITIES[`${path}`]} />
+              </div>
```
Comment (reginawang99) [inline src/components/Activities.tsx:44]: maybe we can just take this out of the map?

File: src/components/shared/ActivityCard.tsx [hunk 3]
```diff
@@ -9,9 +9,9 @@ export interface ActivityCardProps {
 function ActivityCard(props: ActivityCardProps): JSX.Element {
   const {id, description} = props;
```
Comment (reginawang99) [inline src/components/shared/ActivityCard.tsx:10]: maybe we can add a prop for hover

File: src/components/styles/Activities.scss [hunk 4]
```diff
@@ -71,6 +71,15 @@
   background-image: url('../../assets/mind-reading.svg');
 }
 
+#coming-soon {
+  background-image: url('../../assets//coming-soon.svg');
```
Comment (reginawang99) [inline src/components/styles/Activities.scss:75]:
```suggestion
  background-image: url('../../assets/coming-soon.svg');
```

**Review by reginawang99 — CHANGES_REQUESTED** [review]
just nits :PP

File: src/components/Activities.tsx (hunk 1 above)
Comment (darren2hang) [reply src/components/Activities.tsx:24]: Ok!

File: src/components/shared/ActivityCard.tsx (hunk 3 above)
Comment (darren2hang) [reply src/components/shared/ActivityCard.tsx:11]: Is it common practice to store the props in const variables like this?

Comment (reginawang99) [reply src/components/shared/ActivityCard.tsx:11]: mmm this is typescript so yes, but if you were just doing javascript, there's something else called [PropTypes](https://reactjs.org/docs/typechecking-with-proptypes.html). 

it mostly depends on what you want--this helps make the component easier to reuse + prevent errors, but if you didn't really care about clean code you wouldn't i guess haha (hackathons tho)

File: src/components/shared/ActivityCard.tsx [hunk 5]
```diff
@@ -4,12 +4,13 @@ import { capitalize } from '../../utils';
...
   id: string;
   description: string;
+  disableHover?: boolean;
 }
 
 function ActivityCard(props: ActivityCardProps): JSX.Element {
   const {id, description} = props;
   const title = id.split('-').map((word) => capitalize(word)).join(' ');
-  const activityCardStyle = id === 'coming-soon' ? 'activity-card no-hover' : 'activity-card';
+  const activityCardStyle = (props.disableHover !== undefined && props.disableHover === true) ? 'activity-card no-hover' : 'activity-card';
```
Comment (reginawang99) [inline src/components/shared/ActivityCard.tsx:13]:
```suggestion
  const activityCardStyle = (props.disableHover === true) ? 'activity-card no-hover' : 'activity-card';
```

**Review by reginawang99 — CHANGES_REQUESTED** [review]
dot change

File: src/components/shared/ActivityCard.tsx (hunk 3 above)
Comment (darren2hang) [reply src/components/shared/ActivityCard.tsx]: doesn't the interface do the type checking for us though? And from what I know, props are read only. So is the point of this to put a const on it or is it to remind the user what props are available?

Comment (reginawang99) [reply src/components/shared/ActivityCard.tsx]: OnPaste 20210331-190550

Comment (darren2hang) [reply src/components/shared/ActivityCard.tsx]: to me it seems like making an interface achieves the same thing as doing const {..} = props. So is the const {..} = props just a thing to do when there isn't an interface defined.
Or is it just that people in general do both, even though it's not necessary for the code to compile?

**Review by reginawang99 — APPROVED** [review]
LGTM!

**Pull request merged**